src/backend/gl/Cargo.toml

```diff
@@ -23,7 +23,7 @@ bitflags = "1"
 log = { version = "0.4" }
 gfx-hal = { path = "../../hal", version = "0.2" }
 smallvec = "0.6"
-glow = { version = "0.2.0" }
+glow = { git = "https://github.com/grovesNL/glow", rev = "054aabc1bfbad472e4a08bc55912552d8211daee" }
```

Comment:
@grovesNL what's the plan here? Are we waiting for more fixes to `glow` before a patch can be released?

Reply:
Nope, 0.2.0 was published a week ago but there have been a few PRs since then. We can publish whenever this branch is ready to replace the git dependency